gtk4/src/symbolic_paintable.rs

```diff
@@ -8,7 +8,7 @@ pub trait SymbolicPaintableExtManual: 'static {
     #[doc(alias = "gtk_symbolic_paintable_snapshot_symbolic")]
     fn snapshot_symbolic(
         &self,
-        snapshot: &impl IsA<gdk::Snapshot>,
+        snapshot: &gdk::Snapshot,
```

Comment:
Why?

Reply:
We don't do that anywhere else; ideally we would just cast and make the api uses gtk::Snapshot here really